macros/FirefoxOSDefaultAPISidebar.ejs

```diff
@@ -0,0 +1 @@
+REDIRECT <a class="redirect" href="/en-US/docs/Template:FirefoxOSAPISidebar">Template:FirefoxOSDefaultAPISidebar</a>
```

Comment:
[0 pages are calling this macro](https://developer.mozilla.org/en-US/search?locale=*&kumascript_macros=FirefoxOSDefaultAPISidebar&topic=none), so it is fine for this macro to not be imported. For other "redirect macros" that are called by pages, we probably need to create a macro that calls through like `<%- template('Template:HTML:Element_Navigation', arguments) %>`. 
(These are good first candidates for writers/content contributors to clean up once we are done importing. Sigh.)